force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,8 +40,10 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String message = 'Hello, Salesforce!';
+        Integer LengthofString = message.length(); 
+        String substring = message.substring(0, 5); 
+        String lowerCaseString = message.toLowerCase();   
```

Comment:
You got off to a good start. But all of these need to be added to a List of Strings and returned